packages/lib/src/vis/utils.ts

```diff
@@ -34,6 +35,14 @@ export const CAMERA_TOP_RIGHT = new Vector3(1, 1, 0);
 
 const AXIS_OFFSETS = { vertical: 72, horizontal: 40, fallback: 16 };
 
+export const DEFAULT_INTERACTIONS_KEYS: InteractionKeys = {
+  Pan: undefined,
+  Zoom: undefined,
+  XAxisZoom: 'Alt',
+  YAxisZoom: 'Shift',
+  SelectToZoom: 'Control',
+};
```

Comment:
Just a thought: I wonder whether `{ Pan: true }` would feel like a more natural API. I'm not a huge fan of relying on the fact that the property `Pan` is present in the array even though it is set to `undefined`. When we start passing objects with `modifierKey` and `disabled`, it may still be convenient to allow passing `Pan: true` as a shortcut for `Pan: {}` (i.e. enabled with no modifier key).